src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -1,12 +1,12 @@
 package com.minecolonies.coremod.items;
 
-import com.minecolonies.coremod.blocks.*;
 import net.minecraft.item.Item;
 import net.minecraftforge.registries.IForgeRegistry;
 
 /**
  * Class handling the registering of the mod items.
  */
+@SuppressWarnings("squid:S1444")
 public final class ModItems
 {
     public static Item supplyChest;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make supplyChest a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)